diff -up bochs-2.6.8/iodev/devices.cc.unresolved~ bochs-2.6.8/iodev/devices.cc
--- bochs-2.6.8/iodev/devices.cc.unresolved~	2017-01-14 03:07:56.087450187 +0100
+++ bochs-2.6.8/iodev/devices.cc	2017-01-14 03:09:49.938098956 +0100
@@ -180,6 +180,7 @@ void bx_devices_c::init(BX_MEM_C *newmem
     PLUG_load_plugin(pci, PLUGTYPE_CORE);
     PLUG_load_plugin(pci2isa, PLUGTYPE_CORE);
 #if BX_SUPPORT_PCIUSB
+    PLUG_load_plugin(usb_common, PLUGTYPE_CORE);
     if (chipset == BX_PCI_CHIPSET_I440FX) {
       // UHCI is a part of the PIIX3, so load / enable it
       if (!PLUG_device_present("usb_uhci")) {
@@ -188,8 +189,11 @@ void bx_devices_c::init(BX_MEM_C *newmem
       SIM->get_param_bool(BXPN_UHCI_ENABLED)->set(1);
     }
     usb_enabled = is_usb_enabled();
-    if (usb_enabled)
+    if (!usb_enabled) {
       PLUG_load_plugin(usb_common, PLUGTYPE_CORE);
+      PLUG_unload_plugin(usb_common);
+      BX_PANIC(("Bochs has failed to load usb_uhci???"));
+    }
 #endif
     if (chipset == BX_PCI_CHIPSET_I440FX) {
       PLUG_load_plugin(acpi, PLUGTYPE_STANDARD);
