--- bochs-2.4.5/cpu/fetchdecode64.cc.compile~	2010-11-26 23:25:52.511187671 +0100
+++ bochs-2.4.5/cpu/fetchdecode64.cc	2010-11-26 23:26:03.506187672 +0100
@@ -429,7 +429,7 @@ static const BxOpcodeInfo_t BxOpcodeInfo
   /* 0F 0D /wr */ { 0, BX_IA_PREFETCHW },       // 3DNow! PREFETCH on AMD, NOP on Intel
   /* 0F 0E /wr */ { 0, BX_IA_FEMMS },           // 3DNow! FEMMS
 #if BX_SUPPORT_3DNOW
-  /* 0F 0F /wr */ { BxImmediate_Ib, BX_IA_ERROR, Bx3DNowOpcodeInfo },
+  /* 0F 0F /wr */ { BxImmediate_Ib, BX_IA_ERROR },
 #else
   /* 0F 0F /wr */ { 0, BX_IA_ERROR },
 #endif
@@ -948,7 +948,7 @@ static const BxOpcodeInfo_t BxOpcodeInfo
   /* 0F 0D /dr */ { 0, BX_IA_PREFETCHW },       // 3DNow! PREFETCH on AMD, NOP on Intel
   /* 0F 0E /dr */ { 0, BX_IA_FEMMS },           // 3DNow! FEMMS
 #if BX_SUPPORT_3DNOW
-  /* 0F 0F /dr */ { BxImmediate_Ib, BX_IA_ERROR, Bx3DNowOpcodeInfo },
+  /* 0F 0F /dr */ { BxImmediate_Ib, BX_IA_ERROR },
 #else
   /* 0F 0F /dr */ { 0, BX_IA_ERROR },
 #endif
@@ -1467,7 +1467,7 @@ static const BxOpcodeInfo_t BxOpcodeInfo
   /* 0F 0D /qr */ { 0, BX_IA_PREFETCHW },       // 3DNow! PREFETCH on AMD, NOP on Intel
   /* 0F 0E /qr */ { 0, BX_IA_FEMMS },           // 3DNow! FEMMS
 #if BX_SUPPORT_3DNOW
-  /* 0F 0F /qr */ { BxImmediate_Ib, BX_IA_ERROR, Bx3DNowOpcodeInfo },
+  /* 0F 0F /qr */ { BxImmediate_Ib, BX_IA_ERROR },
 #else
   /* 0F 0F /qr */ { 0, BX_IA_ERROR },
 #endif
@@ -1992,7 +1992,7 @@ static const BxOpcodeInfo_t BxOpcodeInfo
   /* 0F 0D /wm */ { 0, BX_IA_PREFETCHW },       // 3DNow! PREFETCH on AMD, NOP on Intel
   /* 0F 0E /wm */ { 0, BX_IA_FEMMS },           // 3DNow! FEMMS
 #if BX_SUPPORT_3DNOW
-  /* 0F 0F /wm */ { BxImmediate_Ib, BX_IA_ERROR, Bx3DNowOpcodeInfo },
+  /* 0F 0F /wm */ { BxImmediate_Ib, BX_IA_ERROR },
 #else
   /* 0F 0F /wm */ { 0, BX_IA_ERROR },
 #endif
@@ -2511,7 +2511,7 @@ static const BxOpcodeInfo_t BxOpcodeInfo
   /* 0F 0D /dm */ { 0, BX_IA_PREFETCHW },       // 3DNow! PREFETCH on AMD, NOP on Intel
   /* 0F 0E /dm */ { 0, BX_IA_FEMMS },           // 3DNow! FEMMS
 #if BX_SUPPORT_3DNOW
-  /* 0F 0F /dm */ { BxImmediate_Ib, BX_IA_ERROR, Bx3DNowOpcodeInfo },
+  /* 0F 0F /dm */ { BxImmediate_Ib, BX_IA_ERROR },
 #else
   /* 0F 0F /dm */ { 0, BX_IA_ERROR },
 #endif
@@ -3030,7 +3030,7 @@ static const BxOpcodeInfo_t BxOpcodeInfo
   /* 0F 0D /qm */ { 0, BX_IA_PREFETCHW },       // 3DNow! PREFETCH on AMD, NOP on Intel
   /* 0F 0E /qm */ { 0, BX_IA_FEMMS },           // 3DNow! FEMMS
 #if BX_SUPPORT_3DNOW
-  /* 0F 0F /qm */ { BxImmediate_Ib, BX_IA_ERROR, Bx3DNowOpcodeInfo },
+  /* 0F 0F /qm */ { BxImmediate_Ib, BX_IA_ERROR },
 #else
   /* 0F 0F /qm */ { 0, BX_IA_ERROR },
 #endif
--- bochs-2.4.5/cpu/fetchdecode.cc.compile~	2010-11-26 23:25:52.513187671 +0100
+++ bochs-2.4.5/cpu/fetchdecode.cc	2010-11-26 23:26:03.509187672 +0100
@@ -474,7 +474,7 @@ static const BxOpcodeInfo_t BxOpcodeInfo
   /* 0F 0D /wr */ { 0, BX_IA_PREFETCHW },       // 3DNow! PREFETCH on AMD, NOP on Intel
   /* 0F 0E /wr */ { 0, BX_IA_FEMMS },           // 3DNow! FEMMS
 #if BX_SUPPORT_3DNOW
-  /* 0F 0F /wr */ { BxImmediate_Ib, BX_IA_ERROR, Bx3DNowOpcodeInfo },
+  /* 0F 0F /wr */ { BxImmediate_Ib, BX_IA_ERROR },
 #else
   /* 0F 0F /wr */ { 0, BX_IA_ERROR },
 #endif
@@ -1025,7 +1025,7 @@ static const BxOpcodeInfo_t BxOpcodeInfo
   /* 0F 0D /dr */ { 0, BX_IA_PREFETCHW },       // 3DNow! PREFETCH on AMD, NOP on Intel
   /* 0F 0E /dr */ { 0, BX_IA_FEMMS },           // 3DNow! FEMMS
 #if BX_SUPPORT_3DNOW
-  /* 0F 0F /dr */ { BxImmediate_Ib, BX_IA_ERROR, Bx3DNowOpcodeInfo },
+  /* 0F 0F /dr */ { BxImmediate_Ib, BX_IA_ERROR },
 #else
   /* 0F 0F /dr */ { 0, BX_IA_ERROR },
 #endif
@@ -1583,7 +1583,7 @@ static const BxOpcodeInfo_t BxOpcodeInfo
   /* 0F 0D /wm */ { 0, BX_IA_PREFETCHW },       // 3DNow! PREFETCH on AMD, NOP on Intel
   /* 0F 0E /wm */ { 0, BX_IA_FEMMS },           // 3DNow! FEMMS
 #if BX_SUPPORT_3DNOW
-  /* 0F 0F /wm */ { BxImmediate_Ib, BX_IA_ERROR, Bx3DNowOpcodeInfo },
+  /* 0F 0F /wm */ { BxImmediate_Ib, BX_IA_ERROR },
 #else
   /* 0F 0F /wm */ { 0, BX_IA_ERROR },
 #endif
@@ -2134,7 +2134,7 @@ static const BxOpcodeInfo_t BxOpcodeInfo
   /* 0F 0D /dm */ { 0, BX_IA_PREFETCHW },       // 3DNow! PREFETCH on AMD, NOP on Intel
   /* 0F 0E /dm */ { 0, BX_IA_FEMMS },           // 3DNow! FEMMS
 #if BX_SUPPORT_3DNOW
-  /* 0F 0F /dm */ { BxImmediate_Ib, BX_IA_ERROR, Bx3DNowOpcodeInfo },
+  /* 0F 0F /dm */ { BxImmediate_Ib, BX_IA_ERROR },
 #else
   /* 0F 0F /dm */ { 0, BX_IA_ERROR },
 #endif
